### PR TITLE
1890 - add size support to Userpic

### PR DESCRIPTION
**Pull request by relativityboy**
https://github.com/steemit/condenser/issues/1890
Putting back size support using new imagehoster urls.

### CODE REVIEW

File: src/app/components/elements/Userpic.jsx [hunk 1]
```diff
@@ -3,26 +3,41 @@ import { connect } from 'react-redux';
 import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
 import { imageProxy } from 'app/utils/ProxifyUrl';
 
+export const SIZE_SMALL = 'small';
+export const SIZE_MED = 'medium';
+export const SIZE_LARGE = 'large';
+
+const sizeList = [SIZE_SMALL, SIZE_MED, SIZE_LARGE]
+
+export const avatarSize = {
```
Comment (bnchdrff) [inline src/app/components/elements/Userpic.jsx]: what's this for? it's not referenced anywhere

Comment (relativityboy) [reply src/app/components/elements/Userpic.jsx]: It's for `<Userpic size={avatarSize.small} />`

Comment (bnchdrff) [reply src/app/components/elements/Userpic.jsx]: wanna roll that usage into this branch so we can start using it & save some kb/pageload right off the bat?

Comment (relativityboy) [reply src/app/components/elements/Userpic.jsx]: Bring it on in.

Comment (bnchdrff) [reply src/app/components/elements/Userpic.jsx]: https://github.com/steemit/condenser/pull/1909/commits/4f9b508423f9f79e8fe97b74574ce44b8185f042 < - brought it in

**Pull request merged**